server.go

```diff
@@ -2132,23 +2132,26 @@ func (s *server) peerHandler(ctx context.Context) {
 		},
 	}
 
-	if !cfg.DisableDNSSeed {
+	if !cfg.DisableSeeders {
 		// Add peers discovered through DNS to the address manager.
-		params := s.chainParams
-		seeds := make([]string, 0, len(params.DNSSeeds))
-		for _, seed := range params.DNSSeeds {
-			seeds = append(seeds, seed.Host)
-		}
-		defaultPort, _ := strconv.Atoi(params.DefaultPort)
-		connmgr.SeedFromDNS(seeds, uint16(defaultPort), defaultRequiredServices,
-			dcrdLookup, func(addrs []*wire.NetAddress) {
-				// Bitcoind uses a lookup of the dns seeder here. This
-				// is rather strange since the values looked up by the
-				// DNS seed lookups will vary quite a lot.
-				// to replicate this behaviour we put all addresses as
-				// having come from the first one.
-				s.addrManager.AddAddresses(addrs, addrs[0])
-			})
+		seeds := s.chainParams.Seeders()
+		for _, seed := range seeds {
+			host := seed
```

Comment:
No need to do this.  You can just pass seed directly to the goroutine since the param is a copy.